src/Shared/E2ETesting/selenium-config.json

```diff
@@ -1,8 +0,0 @@
-{
-  "drivers": {
-    "chrome": {
-      "version" : "114.0.5735.90"
```

Comment:
In theory we no longer have to configure this or keep updating it manually.

We **do** still have to update `SeleniumWebDriverVersion` in `Versions.props` on a regular cadence, since even though it can acquire newer ChromeDriver versions on its own, it still has to be an up-to-date `Selenium.WebDriver` NuGet package in order to support those newer versions.